langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -152,6 +288,27 @@ public isolated function slice(Type[] arr, int startIndex, int endIndex = arr.le
 # The parameter `func` is called for each member of `arr` in order unless and until a call returns false.
 # When the array is empty, returns true.
 #
+# ```ballerina
+# int[] numbers = [1, 2, 3, 5];
+# boolean allEvenNumbers = numbers.every((number) => number % 2 == 0);
```

Comment:
Wouldn't it be better to use the variable name as something like `hasOnlyEvenNumbers` or, `isAllEvenNumbers`?

Reply:
We use `is`/`has` as a prefix generally with the function/method as opposed to the variable. Is the current form not clear?

Reply:
Since the type of the variable is `boolean`, it would be better to indicate it as a flag variable.
Something like: `evenNumbersFlag` or `allEvenNumbersFlag`. 
IMO, the current form would convey the idea of **an array consisting of all the even numbers**.

Reply:
Given that this is the variable declaration and that it is a boolean variable, I don't think it will be misinterpreted to be an array.